src/bin/analyzer.js

```diff
@@ -81,10 +86,29 @@ let {
   logLevel,
   open: openBrowser,
   exclude: excludeAssets,
+  info,
   args: [bundleStatsFile, bundleDir]
 } = program;
 const logger = new Logger(logLevel);
 
+const packageInfo = {
+  node: 'node -v',
+  npm: 'npm -v',
+  yarn: 'yarn -v',
+  webpack: 'webpack-cli -v',
```

Comment:
What if `webpack-cli` is not installed globally?

Reply:
And even if it's installed globally, it may not match the webpack version used in the user's project.